Makefile

```diff
@@ -50,7 +50,10 @@ build:
 clean:
 	jbuilder clean
 
-.PHONY: build clean
+reindent:
+	ocp-indent --inplace **/*.ml*
```

Comment:
It might be safer to only indent files that Git knows about:
```
git ls-files '*.ml*' '**/*.ml*' | xargs ocp-indent --inplace
```

Reply:
How so? Should we care about files not in source control?

Reply:
Because the command updates files in place and if something goes wrong, you would lose a file that is not under source control. While the danger is small, I think it is safer to limit the scope of a desctructive update.